var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -68,6 +68,7 @@ class PyTorch(PythonPackage, CudaPackage):
 
     variant('cuda', default=True, description='Build with CUDA')
     variant('cudnn', default=True, description='Enables the cuDNN build')
+    variant('rocm', default=False, description='Build with ROCm build')
```

Comment:
The way this is currently implemented, `py-torch +rocm` still depends on CUDA. Is that a reasonable way for this to be implemented?

Reply:
The alternative is to add a conflict so `py-torch +rocm +cuda` is invalid.

Reply:
Added the conflicts('+rocm', when '+cuda')